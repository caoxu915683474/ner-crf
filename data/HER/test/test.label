inform
other
inform
inform
modify
inform
inform
inform
other
inform
inform
inform
ack
inform
inform
inform
inform
other
inform
inform
hello
other
inform
inform
other
inform
inform
hello
inform
inform
inform
other
hello
inform
other
inform
hello
inform
inform
inform
inform
inform
other
inform
other
inform
inform
inform
inform
inform
hello
inform
inform
inform
modify
inform
inform
inform
inform
inform
other
inform
other
inform
inform
inform
hello
hello
inform
inform
inform
inform
inform
inform
inform
inform
inform
inform
other
other
hello
inform
inform
other
hello
inform
inform
other
other
inform
other
inform
inform
inform
other
inform
inform
inform
inform
other
inform
hello
inform
inform
negate
inform
inform
other
inform
inform
inform
inform
other
hello
inform
other
inform
inform
hello
hello
other
inform
other
inform
inform
inform
inform
other
hello
inform
inform
hello
hello
