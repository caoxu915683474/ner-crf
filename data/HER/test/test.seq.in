请 问 您 想 多 长 时 间 使 用 会 议 室 呢 ？ 一 个 小 时
请 问 您 想 预 订 几 个 人 的 会 议 室 呢 ？ 最 大 的 会 议 室 多 大
请 告 诉 her 您 想 预 订 几 月 几 号 的 会 议 室 呢 ？ 后 天 吧
请 问 您 想 多 长 时 间 使 用 会 议 室 呢 ？ 5 分 钟
" 不 对 , 是 9 人 "
请 问 您 有 几 个 人 参 会 呢 ？ 黑 客 松 会 议 室
上 午
参 会 人 员 有   我   林 炜 翔   赵 瑞 辉
请 告 诉 her 您 想 预 订 几 月 几 号 的 会 议 室 呢 ？ 傻 逼
" 12 / 22 ， 9 : 00 am ,   10 人 "
只 有 10
6
嗯 嗯
请 告 诉 her 您 想 预 订 几 月 几 号 的 会 议 室 呢 ？ 明 天
请 告 诉 her 您 想 预 订 几 月 几 号 的 会 议 室 呢 ？ 12 . 10
请 告 诉 her 您 想 预 订 几 月 几 号 的 会 议 室 呢 ？ 9 点 30
四 个 人
刚 才 是 填 的 八 个 人 吗
请 问 您 想 预 订 多 长 时 间 的 会 议 室 呢 ？ 八 九 个 人
请 问 您 有 几 个 人 参 会 呢 ？ 5
你 好 丫
请 问 您 想 几 点 钟 使 用 会 议 室 呢 ？ 请 告 诉 her 您 想 预 订 几 月 几 号 的 会 议 室 呢 ？
请 告 诉 her 你 想 预 订 几 个 人 的 会 议 室 呢 ？ 2 个
半 夜
请 告 诉 her 您 想 预 订 几 月 几 号 的 会 议 室 呢 ？ q
赵 瑞 辉   梁 媛   roy   还 有 炜 翔
请 问 您 有 几 个 人 参 会 呢 ？ 9 : 30 pm
你 好 丫
请 问 参 会 人 员 是 什 么 呢 ？ 炜 翔 、 瑞 辉 、 茹 、 我
请 问 您 想 几 点 钟 使 用 会 议 室 呢 ？ 10 dian
请 告 诉 her 您 想 预 订 几 月 几 号 的 会 议 室 呢 ？ 1 月 1
退 出
你 好
请 问 您 想 预 订 几 点 开 始 的 会 议 室 呢 ？ 15 : 00
请 告 诉 her 您 想 预 订 几 月 几 号 的 会 议 室 呢 ？
请 问 您 想 几 点 钟 使 用 会 议 室 呢 ？ am   3 点
在 不 在 丫
请 问 您 想 几 点 钟 使 用 会 议 室 呢 ？ 今 天 下 午 要 在 大 爆 炸 开 会 ， 8 人
请 告 诉 her 您 想 预 订 几 月 几 号 的 会 议 室 呢 ？ 这 周 五
今 天 下 午 3 点 的 深 度 学 习 会 议 室
晚 上
请 告 诉 her 你 想 预 订 几 个 人 的 会 议 室 呢 ？ 两 个 人 的
请 问 您 有 几 个 人 参 会 呢 ？ 请 问 您 有 几 个 人 参 会 呢 ？
请 问 您 想 几 点 钟 使 用 会 议 室 呢 ？ 3 点 左 右 吧
谢 谢 啦   到 家 说 个
请 告 诉 her 您 想 预 订 几 月 几 号 的 会 议 室 呢 ？ 大 后 天
请 问 您 想 几 点 钟 使 用 会 议 室 呢 ？ 下 午 三 点 钟
何 直 接 ， 孔 以 及 ， 鲁 迅   都 是 参 会 人 员
下 午
请 告 诉 her 您 想 预 订 几 月 几 号 的 会 议 室 呢 ？ 11 月 29 号
请 告 诉 her 您 想 预 订 几 月 几 号 的 会 议 室 呢 ？ 你 好 ！
请 告 诉 her 你 想 预 订 多 长 时 间 的 会 议 室 呢 ？ 1 小 时
请 告 诉 her 您 想 预 订 几 月 几 号 的 会 议 室 呢 ？ 今 天
@ her   时 间 是 2017 - 10 - 10
改 成 火 炬 会 议 室 吧
请 问 您 有 几 个 人 参 会 呢 ？ 希 望 树 会 议 室
请 问 您 想 预 订 多 长 时 间 的 会 议 室 呢 ？ 6 小 时
请 问 您 想 几 点 钟 使 用 会 议 室 呢 ？ 8 点
呆 的 下 午
请 告 诉 her 您 想 预 订 几 月 几 号 的 会 议 室 呢 ？ 10 月 21 日
请 告 诉 her 您 想 预 订 几 月 几 号 的 会 议 室 呢 ？ ekekdk
请 问 您 想 几 月 几 号 使 用 会 议 室 呢 ？ 下 周 一
为 什 么 需 要 等 那 么 就 ？
早 上 22 点
请 告 诉 her 您 想 预 订 几 月 几 号 的 会 议 室 呢 ？ 十 点 十 分
请 问 您 想 预 订 多 长 时 间 的 会 议 室 呢 ？ 半 小 时
hi
會 議 室
请 问 您 想 几 点 钟 使 用 会 议 室 呢 ？ 九 点
请 问 您 有 几 个 人 参 会 呢 ？ 八 个 人
请 问 您 有 几 个 人 参 会 呢 ？ 1
请 问 您 有 几 个 人 参 会 呢 ？ 就 10 人
下 午 9 : 00
请 问 您 想 几 点 钟 使 用 会 议 室 呢 ？ 6
请 问 您 有 几 个 人 参 会 呢 ？ 6
请 告 诉 her 您 想 预 订 几 月 几 号 的 会 议 室 呢 ？ 明 后 天
请 问 您 有 几 个 人 参 会 呢 ？ 一 个
请 问 您 有 几 个 人 参 会 呢 ？ 100 人
请 告 诉 her 您 想 预 订 几 月 几 号 的 会 议 室 呢 ？ hggg
请 告 诉 her 您 想 预 订 几 月 几 号 的 会 议 室 呢 ？ djdjdjdjdjdjddjdjdjdjdjdjdjdj
请 告 诉 her 您 想 预 订 几 月 几 号 的 会 议 室 呢 ？ 你 好
下 午 5 : 30
不 是   我 要 订 深 度 学 习 会 议 室
请 问 您 想 几 点 钟 使 用 会 议 室 呢 ？ 请 告 诉 her 您 想 预 订 几 月 几 号 的 会 议 室 呢 ？
在 吗
请 问 参 会 人 员 是 什 么 呢 ？ 科 比 和 我
请 问 您 想 几 点 钟 使 用 会 议 室 呢 ？ 九 点
请 问 您 有 几 个 人 参 会 呢 ？ 刚 才 是 九 点 吗
请 问 您 有 几 个 人 参 会 呢 ？ 请 问 您 有 几 个 人 参 会 呢 ？
17 - 12 - 20   早 上 七 点 开 会
请 告 诉 her 您 想 预 订 几 月 几 号 的 会 议 室 呢 ？ dhdjdjdj
请 告 诉 her 您 想 预 订 几 月 几 号 的 会 议 室 呢 ？ 9 月 3 号
请 问 您 有 几 个 人 参 会 呢 ？ 12
下 周 三 要 开 会
请 问 您 想 几 个 人 使 用 会 议 室 呢 ？ 久
请 问 您 想 几 点 开 始 使 用 会 议 室 呢 ？ 1500
pm
假 如 能 深 度 学 习 会 议 室 的 话 最 好
11 . 2
请 问 您 想 几 点 钟 使 用 会 议 室 呢 ？ 请 告 诉 her 您 想 预 订 几 月 几 号 的 会 议 室 呢 ？
我 想 预 定 明 天 下 午 三 点 到 五 点 的 聚 光 灯 会 议 室
嗨
下 周 三
请 问 您 有 几 个 人 参 会 呢 ？ 90 个 人
no
请 问 您 有 几 个 人 参 会 呢 ？ 九 个
请 问 您 想 几 点 钟 使 用 会 议 室 呢 ？ 10 :   15 pm
请 告 诉 her 您 想 预 订 几 月 几 号 的 会 议 室 呢 ？ aqqqq
请 问 您 想 多 长 时 间 使 用 会 议 室 呢 ？ 一 小 时
请 告 诉 her 你 想 预 订 多 长 时 间 的 会 议 室 呢 ？ 一 小 时
12
请 告 诉 her 您 想 预 订 几 月 几 号 的 会 议 室 呢 ？ 12 月 一 号
请 告 诉 her 您 想 预 订 几 月 几 号 的 会 议 室 呢 ？ / maintainence
定
请 问 您 有 几 个 人 参 会 呢 ？ 10000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
请 告 诉 her 你 想 预 订 多 长 时 间 的 会 议 室 呢 ？ 张 茹 不 是 设 备
2 小 时
10 : 30
hi ~
你 好 呀
请 告 诉 her 您 想 预 订 几 月 几 号 的 会 议 室 呢 ？ 我 不 想 定 会 议 室
6 点
请 问 您 有 几 个 人 参 会 呢 ？ 请 问 您 有 几 个 人 参 会 呢 ？
请 问 您 想 几 点 开 始 使 用 会 议 室 呢 ？ 下 午 3 点
请 告 诉 her 您 想 预 定 的 参 会 人 员 是 什 么 呢 ？ 七 七 ， 娃 娃 ， 哦 哦 ， 冉 冉 ， 踏 踏
请 问 您 有 几 个 人 参 会 呢 ？ 我 要 订 黑 客 松 会 议 室
请 问 您 有 几 个 人 参 会 呢 ？ 10 个
请 问 您 想 几 点 钟 使 用 会 议 室 呢 ？ 请 告 诉 her 您 想 预 订 几 月 几 号 的 会 议 室 呢 ？
请 问 您 想 几 点 钟 使 用 会 议 室 呢 ？ 你 好
请 告 诉 her 你 想 预 订 几 月 几 号 的 会 议 室 呢 ？ 后 天
请 问 您 想 预 订 几 个 人 的 会 议 室 呢 ？ 100 个
你 好
你 好 哈
