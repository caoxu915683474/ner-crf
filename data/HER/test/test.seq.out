O O O O O O O O O O O O O O O B-DUR I-DUR I-DUR I-DUR
O O O O O O O O O O O O O O O O O O O O O O O
O O O O O O O O O O O O O O O O O O B-DATE I-DATE O
O O O O O O O O O O O O O O O B-DUR I-DUR I-DUR
O O O O O B-PNO I-PNO O
O O O O O O O O O O O B-LOC I-LOC I-LOC O O O
B-APM I-APM
O O O O O O B-PER O B-PER I-PER I-PER O B-PER I-PER I-PER
O O O O O O O O O O O O O O O O O O O O
O B-DATE I-DATE I-DATE O B-TIME I-TIME I-TIME B-APM O O B-PNO I-PNO O
O O B-PNO
B-PNO
O O
O O O O O O O O O O O O O O O O O O B-DATE I-DATE
O O O O O O O O O O O O O O O O O O B-DATE I-DATE I-DATE
O O O O O O O O O O O O O O O O O O B-TIME I-TIME I-TIME
B-PNO I-PNO I-PNO
O O O O O B-PNO I-PNO I-PNO O
O O O O O O O O O O O O O O O O B-PNO I-PNO I-PNO I-PNO
O O O O O O O O O O O B-PNO
O O O
O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O
O O O O O O O O O O O O O O O O O B-PNO I-PNO
B-APM I-APM
O O O O O O O O O O O O O O O O O O O
B-PER I-PER I-PER O B-PER I-PER O B-PER O O O B-PER I-PER
O O O O O O O O O O O B-TIME I-TIME I-TIME B-APM
O O O
O O O O O O O O O O O B-PER I-PER O B-PER I-PER O B-PER O B-PER
O O O O O O O O O O O O O O B-TIME O
O O O O O O O O O O O O O O O O O O B-DATE I-DATE I-DATE
O O
O O
O O O O O O O O O O O O O O O O B-TIME I-TIME I-TIME
O O O O O O O O O O O O O O O O O O
O O O O O O O O O O O O O O B-APM O B-TIME I-TIME
O O O O
O O O O O O O O O O O O O O B-DATE I-DATE B-APM I-APM O O B-LOC I-LOC I-LOC O O O B-PNO I-PNO
O O O O O O O O O O O O O O O O O O O B-DATE I-DATE
B-DATE I-DATE B-APM I-APM B-TIME I-TIME O B-LOC I-LOC I-LOC I-LOC O O O
B-APM I-APM
O O O O O O O O O O O O O O O O O B-PNO I-PNO I-PNO O
O O O O O O O O O O O O O O O O O O O O O O
O O O O O O O O O O O O O O B-TIME I-TIME O O O
O O O O O O O O
O O O O O O O O O O O O O O O O O O B-DATE I-DATE I-DATE
O O O O O O O O O O O O O O B-APM I-APM B-TIME I-TIME I-TIME
B-PER I-PER I-PER O B-PER I-PER I-PER O B-PER I-PER O O O O O O O
B-APM I-APM
O O O O O O O O O O O O O O O O O O B-DATE I-DATE I-DATE I-DATE
O O O O O O O O O O O O O O O O O O O O O
O O O O O O O O O O O O O O O O O O B-DUR I-DUR I-DUR
O O O O O O O O O O O O O O O O O O B-DATE I-DATE
O O O O O O B-DATE I-DATE I-DATE I-DATE I-DATE
O O B-LOC I-LOC O O O O
O O O O O O O O O O O B-LOC I-LOC I-LOC O O O
O O O O O O O O O O O O O O O O B-DUR I-DUR I-DUR
O O O O O O O O O O O O O O B-TIME I-TIME
O O B-APM I-APM
O O O O O O O O O O O O O O O O O O B-DATE I-DATE I-DATE I-DATE
O O O O O O O O O O O O O O O O O O O
O O O O O O O O O O O O O O O B-DATE I-DATE I-DATE
O O O O O O O O O O
B-APM I-APM B-TIME I-TIME
O O O O O O O O O O O O O O O O O O B-TIME I-TIME I-TIME I-TIME
O O O O O O O O O O O O O O O O B-DUR I-DUR I-DUR
O
O O O
O O O O O O O O O O O O O O B-TIME I-TIME
O O O O O O O O O O O B-PNO I-PNO I-PNO
O O O O O O O O O O O B-PNO
O O O O O O O O O O O O B-PNO I-PNO
B-APM I-APM B-TIME I-TIME I-TIME
O O O O O O O O O O O O O O B-TIME
O O O O O O O O O O O B-PNO
O O O O O O O O O O O O O O O O O O O O O
O O O O O O O O O O O B-PNO I-PNO
O O O O O O O O O O O B-PNO I-PNO
O O O O O O O O O O O O O O O O O O O
O O O O O O O O O O O O O O O O O O O
O O O O O O O O O O O O O O O O O O O O
B-APM I-APM B-TIME I-TIME I-TIME
O O O O O O B-LOC I-LOC I-LOC I-LOC O O O
O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O
O O
O O O O O O O O O O O B-PER I-PER O B-PER
O O O O O O O O O O O O O O B-TIME I-TIME
O O O O O O O O O O O O O O B-TIME I-TIME O
O O O O O O O O O O O O O O O O O O O O O O
B-DATE I-DATE I-DATE I-DATE I-DATE O B-APM I-APM B-TIME I-TIME O O
O O O O O O O O O O O O O O O O O O O
O O O O O O O O O O O O O O O O O O B-DATE I-DATE I-DATE I-DATE
O O O O O O O O O O O B-PNO
B-DATE I-DATE I-DATE O O O
O O O O O O O O O O O O O O O
O O O O O O O O O O O O O O O O
B-APM
O O O B-LOC I-LOC I-LOC I-LOC O O O O O O O
B-DATE I-DATE I-DATE
O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O
O O O O B-DATE I-DATE B-APM I-APM B-TIME I-TIME O B-TIME I-TIME O B-LOC I-LOC I-LOC O O O
O
B-DATE I-DATE I-DATE
O O O O O O O O O O O B-PNO I-PNO I-PNO
O
O O O O O O O O O O O B-PNO I-PNO
O O O O O O O O O O O O O O B-TIME I-TIME I-TIME I-TIME B-APM
O O O O O O O O O O O O O O O O O O O
O O O O O O O O O O O O O O O B-DUR I-DUR I-DUR
O O O O O O O O O O O O O O O O O O B-DUR I-DUR I-DUR
O
O O O O O O O O O O O O O O O O O O B-DATE I-DATE I-DATE I-DATE
O O O O O O O O O O O O O O O O O O O O
O
O O O O O O O O O O O B-PNO
O O O O O O O O O O O O O O O O O O B-PER I-PER O O O O
B-DUR I-DUR I-DUR
B-TIME I-TIME I-TIME
O O
O O O
O O O O O O O O O O O O O O O O O O O O O O O O O
B-TIME I-TIME
O O O O O O O O O O O O O O O O O O O O O O
O O O O O O O O O O O O O O O B-APM I-APM B-TIME I-TIME
O O O O O O O O O O O O O O O O O O B-PER I-PER O B-PER I-PER O B-PER I-PER O B-PER I-PER O B-PER I-PER
O O O O O O O O O O O O O O B-LOC I-LOC I-LOC O O O
O O O O O O O O O O O B-PNO I-PNO
O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O
O O O O O O O O O O O O O O O O
O O O O O O O O O O O O O O O O O O B-DATE I-DATE
O O O O O O O O O O O O O O O B-PNO I-PNO
O O
O O O
